### PR TITLE
fix EventQueue storage fragmentation example

### PR DESCRIPTION
**Pull request by maciejbocianski**

### CODE REVIEW

**Comment by AnotherButler** [comment]
ping @jamesbeyond

**Review by jamesbeyond — APPROVED** [review]
Looks Good

File: docs/api/rtos/events_tutorial.md [hunk 1]
```diff
@@ -116,17 +116,21 @@ If your project only uses fix-sized events, you can use a counter that tracks th
 If your projects uses variable-sized events, you can calculate the number of available events of a specific size because successfully allocated memory is never fragmented further. However, untouched space can service any event that fits, which complicates such a calculation.
 
 ```
-EventQueue queue(8*sizeof(int)); // 8 words of storage
-queue.call(func, 1);       // requires 2 words of storage
-queue.call(func, 1, 2, 3); // requires 4 words of storage
+// event size in words: 9 + callback_size + arguments_size (where 9 is internal space for event data)
```
Comment (evedon) [inline docs/api/rtos/events_tutorial.md:119]: @maciejbocianski I reviewed this example and noticed that callback_size is 4 words.
`EVENTS_EVENT_SIZE` is 13 words (9 for equeue_event +4 for mbed::Callback)

Comment (gpsimenos) [reply docs/api/rtos/events_tutorial.md:119]: PR #1339 opened to correct this.

**Pull request merged**